service/src/main/java/com/commercetools/pspadapter/payone/IntegrationService.java

```diff
@@ -53,6 +56,19 @@ public void start() {
 
         Spark.port(port());
 
+        redirect.any("/", "/health", Redirect.Status.MOVED_PERMANENTLY);
```

Comment:
Why it is confusing - that is what health/status endpoints are for? Guess what you get if you access: https://api.twitter.com or https://graph.facebook.com? You might return 400 and provide in response info which endpoints are exposed for communication or return 404